### PR TITLE
Ensure user belongs to libvirt group before any action

### PR DESCRIPTION
**Pull request by jamesregis**
Will make checks to ensure user belongs to libvirt group. It not, molecule will stop with an error message and the solution.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @ssbarnea @jamesregis
```
Comment (goldyfruit) [inline .github/CODEOWNERS:1]: This file should not be part of the PR, PR #23 still in review.

**Review by goldyfruit — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ssbarnea) [reply .github/CODEOWNERS:1]: When he will rebase it will dissaper.

**Review by ssbarnea — APPROVED** [review]

**Review by goldyfruit — APPROVED** [review]
